pennylane_lightning/src/benchmarks/Bench_ApplyOperations.cpp

```diff
@@ -0,0 +1,196 @@
+#include <map>
+#include <random>
+#include <string>
+#include <string_view>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+#include "Constant.hpp"
+#include "StateVectorManaged.hpp"
+
+#include <benchmark/benchmark.h>
+
+using OpName = std::string;
+using NumWires = size_t;
+using NumParams = size_t;
+
+using SerializedOp = std::tuple<OpName, NumWires, NumParams>;
+
+static inline auto getLightningGates()
+    -> std::map<OpName, std::pair<NumWires, NumParams>> {
+    // All gates except multi-qubit-gates
+    namespace Constant = Pennylane::Gates::Constant;
+    std::map<OpName, std::pair<NumWires, NumParams>> all_gates;
+    for (const auto &[gate_op, gate_name] : Constant::gate_names) {
+        if (!Pennylane::Util::array_has_elt(Constant::multi_qubit_gates,
+                                            gate_op)) {
+            all_gates.emplace(
+                gate_name,
+                std::make_pair(static_cast<size_t>(Pennylane::Util::lookup(
+                                   Constant::gate_wires, gate_op)),
+                               static_cast<size_t>(Pennylane::Util::lookup(
+                                   Constant::gate_num_params, gate_op))));
+        }
+    }
+    return all_gates;
+}
+
+static void availableLightningGates(benchmark::State &state) {
+    for (auto _ : state) {
+        auto res = getLightningGates();
+        benchmark::DoNotOptimize(res.size());
+    }
+}
+BENCHMARK(availableLightningGates);
+
```

Comment:
To make this file easier on the eye, maybe adding line-breaking comments to compartmentalize the tests would help:
```
//*************************************************//
```
or 
```
/*#######################################################*/
```

What do you think?

Reply:
Thanks @mlxd for the suggestion. I hope this is fixed now.